### PR TITLE
Add command "mbt merge" for merging mta.yaml with extensions to a new file

### PR DESCRIPTION
**Pull request by tal-sapan**
### Checklist
- [X] Code compiles correctly
- [X] Relevant tests were added (unit / contract / integration)
- [x] Relevant logs were added
- [X] Formatting and linting run locally successfully
- [X] All tests pass
- [ ] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

File: internal/artifacts/meta.go [hunk 1]
```diff
@@ -1,8 +1,12 @@
 package artifacts
 
 import (
+	"fmt"
 	"github.com/pkg/errors"
```
Comment (allaVolkov) [inline internal/artifacts/meta.go]: please fix imports sequence

File: internal/artifacts/meta_test.go [hunk 2]
```diff
@@ -190,6 +191,48 @@ cli_version:["x"]
 			Ω(mtarPath).Should(BeAnExistingFile())
 		})
 	})
+	Describe("ExecuteMerge", func() {
+		It("Succeeds with single mtaext file", func() {
+			resultFileName := "result.yaml"
```
Comment (allaVolkov) [inline internal/artifacts/meta_test.go]: I should move resultFileName out of It's.

Comment (tal-sapan) [reply internal/artifacts/meta_test.go]: Done

File: internal/artifacts/meta.go (hunk 1 above)
Comment (tal-sapan) [reply internal/artifacts/meta.go]: Done

File: internal/artifacts/meta_test.go [hunk 3]
```diff
@@ -190,6 +191,46 @@ cli_version:["x"]
...
+			Ω(err).Should(Succeed())
+			Ω(resultFilePath).Should(BeAnExistingFile())
+			compareMTAContent(getTestPath("mta_with_ext", "golden2.yaml"), resultFilePath)
+		})
+		It("Fails when the result file name is not sent", func() {
+			err := ExecuteMerge(getTestPath("mta_with_ext"), getResultPath(), []string{"cf-mtaext.yaml"}, "", os.Getwd)
+			checkError(err, mergeNameRequiredMsg)
+		})
+		It("Fails when the result file already exists", func() {
+			resultFilePath := resultFilePath
```
Comment (allaVolkov) [inline internal/artifacts/meta_test.go]: remove please

Comment (tal-sapan) [reply internal/artifacts/meta_test.go]: Done

**Comment by tal-sapan** [comment]
> Coverage decreased by 0.3%. Can you please decrease this decrease?

The only new code that isn't covered is uncommon error paths (os.Getwd, yaml.Marshal) and calling the new command itself (there's an integration test for that but it doesn't increase the coverage)

**Comment by tal-sapan** [comment]
> Coverage decreased by 0.3%. Can you please decrease this decrease?

Added tests

File: cmd/merge_test.go [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"path/filepath"
+)
+
+var _ = Describe("Merge commands call", func() {
+	AfterEach(func() {
+		dir, _ := os.Getwd()
```
Comment (allaVolkov) [inline cmd/merge_test.go]: you can use getTestPath(...)

File: cmd/merge_test.go [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+		Ω(os.RemoveAll(filepath.Join(dir, "testdata", "mtaext", "result.yaml"))).Should(Succeed())
+	})
+	It("merges with one extension", func() {
+		dir, _ := os.Getwd()
+		path := filepath.Join(dir, "testdata", "mtaext")
```
Comment (allaVolkov) [inline cmd/merge_test.go]: you can use getTestPath(...)

File: cmd/merge_test.go (hunk 4 above)
Comment (tal-sapan) [reply cmd/merge_test.go]: Done

File: cmd/merge_test.go (hunk 5 above)
Comment (tal-sapan) [reply cmd/merge_test.go]: Done

**Pull request merged**